portfolio/src/main/java/com/google/sps/servlets/NicknameServlet.java

```diff
@@ -0,0 +1,83 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Optional;
+
+@WebServlet("/nickname")
+public class NicknameServlet extends HttpServlet {
+  private static final String NICKNAME = "nickname";
+  private static final String ID = "id";
+  private static final String USER_INFO = "userInfo";
+
+  private static final String BOTTOM_OF_PAGE = "/index.html#connect-container";
+  
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+    PrintWriter out = response.getWriter();
+    out.println("<h1>Set Nickname</h1>");
+
+    UserService userService = UserServiceFactory.getUserService();
```

Comment:
Do you want to initialize this in the ```init()``` method?

Reply:
Yes, I think that would be a good idea. Thanks! I added an init() for DataServlet as well because datastore is used in both methods.

Done!